django_guid/signals.py

```diff
@@ -0,0 +1,26 @@
+import logging
+from typing import Optional
+
+from django.core.signals import request_finished
+from django.dispatch import receiver
+
+from .middleware import GuidMiddleware
```

Comment:
Yes I think this might also fix import problems and let you import version to setup.py again. It does seem like the `proper` way of doing it, though I agree that having to add the package to your installed apps just for this change is annoying. 

¯\_(ツ)_/¯ 

Feel free to implement the version you prefer.